scripts/gh-projects/examples/mux-video-integration/phase-2/c1.md

```
@@ -0,0 +1,10 @@
+Add a `.env.example` at the repo root documenting the Mux Data env key.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)